openapi.json

```diff
@@ -0,0 +1,282 @@
+{
```

Comment:
this is generated output just for swagger or does fastapi actual consume it to define/validate the route response and request bodies?

if generated just for swagger it should not be committed... add to README how to generate it

Reply:
It was just for swagger. Its purpose was to allow users learn the API endpoint. The generation guid is added to the readme.